### DISCUSSION TITLE
Add the Esc key to keypress events.

### DISCUSSION THREAD

**Discussion by PARTOVIY** [SmartlyDressedGames/Unturned-3.x-Community#3669] [Ideas]
Nelson, users really need this feature. It is very difficult for them to press a separate UI button to exit the "menu" and other UI interfaces.

**Comment by XanderCodes** [comment]
This would be amazing. The amount of time I go to press ESC out of muscle memory when using UI's is incredible.

**Reply by sunnamed434** [reply]
Interesting
